shared/translations/languages/de.json

```diff
@@ -1821,7 +1821,8 @@
     "USD": "USD",
     "How many {{token}} tokens to borrow?": "Wie viele {{token}} Token möchtest du ausleihen?",
     "Your vault needs at least 50% of DFI and/or DUSD as collateral": "Dein Vault braucht mindestens 50 % DFI und/oder DUSD als Besicherung",
-    "This vault needs at least 50% of DFI and/or DUSD to be available for use in minting dTokens": "Dieser Vault braucht mindestens 50% DFI und/oder DUSD, um für das Minting von dToken zur Verfügung zu stehen"
+    "This vault needs at least 50% of DFI and/or DUSD to be available for use in minting dTokens": "Dieser Vault braucht mindestens 50% DFI und/oder DUSD, um für das Minting von dToken zur Verfügung zu stehen",
+    "Insufficient DFI in vault. Add more to borrow DUSD.": "Unzureichende DFI im Vault. Füge mehr hinzu, um DUSD zu leihen."
```

Comment:
instead of adding new entry, can we replace the old string? 
same for other `.json`

Reply:
updated the PR